lib/ansible/modules/windows/win_pagefile.ps1

```diff
@@ -91,22 +90,20 @@ if ($state -eq "absent") {
     }
 
     # Make sure drive is accessible
-    if (($test_path) -and (-not (Test-Path "${drive}:"))) {
+    if (($testPath) -and (-not (Test-Path "${drive}:"))) {
```

Comment:
This looks like it should be in it's own PR with a changelog and subsequently backported.